### PR TITLE
Add readme file

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?

It adds a readme to the repository.

#### How can this be tested?
Navigate to this 
https://github.com/crane-cloud/documentation/tree/fx-add-readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+#### Install and creating a virtual environment run
+ 
+     > `python3 -m pip install virtualenv`
+
+#### To create a virtual environment run
+
+     > `python3 -m virtualenv name-of-your-virtual-environment`
+
+**NOTE** Remeber to add the virtual environment to your gitignore.
```
Comment (MutegekiHenry) [inline README.md:21]: @neelxie Just a typo to correct *Remember

**Review by MutegekiHenry — COMMENTED** [review]
Just a small typo to update.

File: README.md (hunk 1 above)
Comment (Mubangizi) [reply README.md:21]: Yeah apart from this it looks good to me

Comment (neelxie) [reply README.md]: @MutegekiHenry @Mubangizi this has been fixed.

Comment (MutegekiHenry) [reply README.md]: looks good

**Review by MutegekiHenry — APPROVED** [review]
All good.

**Review by Mubangizi — APPROVED** [review]
LGTM

**Pull request merged**